PLAN.md

```
@@ -0,0 +1,746 @@
+ALTER TABLE social_posts ADD COLUMN ai_provider VARCHAR(50);        -- which AI generated it
+ALTER TABLE social_posts ADD COLUMN ai_generated BOOLEAN DEFAULT false;
+ALTER TABLE social_posts ADD COLUMN ai_prompt_used TEXT;
+ALTER TABLE social_posts ADD COLUMN ai_variations JSONB;
+ALTER TABLE social_posts ADD COLUMN language VARCHAR(10) DEFAULT 'en';  -- 'en', 'lo', 'th', 'fr', etc.
+ALTER TABLE social_posts ADD COLUMN bitly_url TEXT;                 -- shortened URL
+ALTER TABLE social_posts ADD COLUMN bitly_clicks INTEGER DEFAULT 0; -- cached click count
+ALTER TABLE social_posts ADD COLUMN pixel_events JSONB;             -- tracking pixel data
+```
+
+### Environment Variables Needed
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)